src/site/_includes/js/eligibility.js

```diff
@@ -597,6 +602,16 @@ function onChangeSpouse(event) {
       }
     }
   }
+
+  if (event.target.checked) {
+    event.target.dependentParameter.checked = false;
```

Comment:
Oops, sorry I meant to comment on this earlier.  Can you put this inside the existing `if(event.target.checked)` block?

Reply:
Done, see latest commit